libs/blocks/chart/chart.js

```diff
@@ -119,6 +122,30 @@ async function fetchData(link) {
     data.data = json.data;
     data.series = [];
   }
+  return data;
+}
+
+export function listChartData(json) {
+  const data = {};
+
+  if (json[':type'] === 'multi-sheet') {
```

Comment:
Based on what we talked about, I thought the list chart excel always had to be a multisheet

Reply:
We have to use the first header to pull the Title, and there are no options for ordered/numbered list but it works and then authors don't have to add a metadata table.
And for multisheet if they don't have a metadata table we can use that same first header.

Reply:
We need an option for ordered lists, so I'm not sure I understand how that works.

Generally, I don't think we should extra code to accommodate different ways of authoring. There should just be the single way it's supposed to be authored

Reply:
Take a look a the three ListChart examples in drafts/data-viz/, in ListChartCarouselMetadata.xlsx if you have a sheet called helix-table that is where you set an ordered list. 

That sounds counter to the authoring guides for Helix, do you want me to remove the accommodations? How are authors going to know they need a special table?

Reply:
In my opinion, the confusion of having two different ways to author the same thing outweighs any gains of the second way being simpler. It also doesn't seem worth the extra code.

But I'd be interested to hear what the milo team thinks about the authoring experience. @chrischrischris @auniverseaway

Reply:
In general I don't think we should provide different ways to author the same thing.  Some things can be flexible like capitalization of properties.

In this case it sounds like it makes the authoring easier by not having to create multiple files and instead using a single file with multiple sheets?  I think there's value to that as long as we communicate it clearly to authors.

Reply:
Please look at the example spreadsheets, each one represents a different block not multiple files for one block. It's really not that different from the authoring perspective and add a nice progression for adding data.
1. Create a workbook and add my data -> works without ordered list option.
2. Add a second sheet, rename them with helix-  and add my data -> works without ordered list option.
3. Add a third sheet named helix-table, add table of contents and options, -> works with the ordered list option.

Otherwise you have to create a workbook with two sheets, one has to be named helix-table, and hast to be in the right format or you get nothing. And we're talking about 10 lines of code.